src/System.Net.Http.WinHttpHandler/tests/UnitTests/WinHttpResponseHeaderReaderTest.cs

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Justin Van Patten. All rights reserved.
```

Comment:
@stephentoub This differs from the usual copyright headers.  What is the policy on this?